controllers/authorino_controller.go

```diff
@@ -424,9 +429,39 @@ func (r *AuthorinoReconciler) authorinoDeploymentChanges(existingDeployment, des
 	return false
 }
 
+func authorinoServiceAccountName(crName string) string {
```

Comment:
Should we consider moving this function to the resources package under the `k8s_rbac.go` in case it needs to be used somewhere else in the code? https://github.com/Kuadrant/authorino-operator/blob/main/pkg/resources/k8s_rbac.go

Reply:
Some names are defined in  https://github.com/Kuadrant/authorino-operator/blob/main/pkg/resources/k8s_utils.go, while rbac, service, and deployment seem to be only for building basic structs based on k8s types.

Should I add it to rbac or to utils? Or maybe have a new file only for naming objects?

Reply:
True, what do you think if we add this function to utils, make it accessible only in the package then create a method in k8s_rbac.go that receives the name and namespace of the service account and return a service account's object with these attributes assigned?

Reply:
Done.